### PR TITLE
Ports ethereals from TG

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

See title. Ethereals are a unique species that light up their environment with their chosen color, lose light as they get hurt, and require power instead of nutrition. Living lightbulbs, in other words.

## Why It's Good For The Game

I like unique species.

## Changelog
:cl:
add: Ethereals
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/ethereal.dm [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+/datum/species/ethereal
+	name = "Ethereal"
+	id = "ethereal"
+	attack_verb = "burn"
+	attack_sound = 'sound/weapons/etherealhit.ogg'
+	miss_sound = 'sound/weapons/etherealmiss.ogg'
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/ethereal
+	mutantstomach = /obj/item/organ/stomach/ethereal
+	mutanttongue = /obj/item/organ/tongue/ethereal
+	exotic_blood = /datum/reagent/consumable/liquidelectricity //Liquid Electricity. fuck you think of something better gamer
```
Comment (timothyteakettle) [inline code/modules/mob/living/carbon/human/species_types/ethereal.dm:12]: this is uh way too strong, change it to a weakened subtype or something not as powergamey

File: code/modules/mob/living/carbon/human/species_types/ethereal.dm [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+	sexes = FALSE
+	toxic_food = NONE
+	/*
+	citadel doesn't have per-species temperatures, yet
+	// Body temperature for ethereals is much higher then humans as they like hotter environments
+	bodytemp_normal = (BODYTEMP_NORMAL + 50)
+	bodytemp_heat_damage_limit = FIRE_MINIMUM_TEMPERATURE_TO_SPREAD // about 150C
+	// Cold temperatures hurt faster as it is harder to move with out the heat energy
+	bodytemp_cold_damage_limit = (T20C - 10) // about 10c
+	*/
```
Comment (timothyteakettle) [inline code/modules/mob/living/carbon/human/species_types/ethereal.dm:29]: a good placeholder is to use what lizards and slimes have, which is a heat and cold damage modifier, also slimes have special lungs that take damage for breathing cold air earlier, too

**Comment by Ludox235** [comment]
Make moths get a good moodlet from hugging them (I hugged a lightbulb)

**Comment by tiramisuapimancer** [comment]
there aren't moths anymore

**Comment by silicons** [comment]
GIVE US MOTHS BACK

File: code/modules/mob/living/carbon/human/species_types/ethereal.dm (hunk 1 above)
Comment (silicons) [reply code/modules/mob/living/carbon/human/species_types/ethereal.dm:12]: nerf it when needed tbh

**Pull request merged**